components/VC/Views/VCCardView.tsx

```diff
@@ -54,10 +47,34 @@ export const VCCardView: React.FC<VCItemProps> = props => {
     });
   }, [verifiableCredentialData?.wellKnown]);
 
-  if (!isVCLoaded(credential, fields)) {
+  if (!isVCLoaded(controller.credential, fields)) {
     return <VCCardSkeleton />;
   }
 
+  const VCCardViewContentFn = props => (
```

Comment:
Can we name it as VCCardViewContent itself rather than VCCardViewContentFn if its not conflicting?

Reply:
it is creating the conflict, first I have used VCCardViewContent for the method. The credentials are not being downloaded and the card is stucking in load vc state and if we close and reopen the app it is showing the error whilw downloading and deleting the downloaded vc